tracim/tracim/model/data.py

```diff
@@ -544,6 +546,7 @@ class ContentRevisionRO(DeclarativeBase):
     )
     file_mimetype = Column(Unicode(255),  unique=False, nullable=False, default='')
     file_content = deferred(Column(LargeBinary(), unique=False, nullable=True))
+    depot_file = Column(UploadedFileField, unique=False, nullable=True)
```

Comment:
Add a comment with a link to documentation on howto manipulate depot_file (as the interface is not homogeneous for setting and getting value)